### PR TITLE
[Author] Add slim.js via single package.json w/ npm auto-update

### PR DESCRIPTION
**Pull request by eavichay**
Pull request for issue: #11007 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@eavichay Thanks! We may need to add some path to `.gitignore` to avoid auto-adding the version which can't run on cdn( As you said, we need to ignore versions before 2.6.x, right?). Do you mind if I add a new commit to add it following with your commit?

**Comment by eavichay** [comment]
I’d appreciate your help.

Can’t wait until this is merged and CDN’ed :)

**Comment by maruilian11** [comment]
@eavichay ok. I will add it later.

**Comment by maruilian11** [comment]
@PeterDaveHello I thinks this PR is fine now.

**Comment by PeterDaveHello** [comment]
@maruilian11 PR title, again.

**Comment by PeterDaveHello** [comment]
@maruilian11 why don't you also fix commit message of https://github.com/cdnjs/cdnjs/pull/11027/commits/2359447f839b594248db111afe87a07b445e248a?

File: ajax/libs/slim.js/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "slim.js",
+  "filename": "Slim.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/slim.js/package.json]: Looks like I can't use uglifyjs to minify Slim.js, @eavichay can you please take a look?

**Comment by eavichay** [comment]
uglify js harmomy does the trick

**Comment by PeterDaveHello** [comment]
@eavichay really? I tested with uglify-js v2.8.21 but it doesn't work

**Comment by eavichay** [comment]
would you rather me provide the minified or just host the regular one...

**Comment by PeterDaveHello** [comment]
@eavichay it'll be very good if you can provide one, would you?

**Comment by eavichay** [comment]
[Slim.min.js.gz](https://github.com/cdnjs/cdnjs/files/907764/Slim.min.js.gz)

**Comment by PeterDaveHello** [comment]
@eavichay thanks a lot, so would you publish it in future releases just in the repo? Can you also let me know the details about the process you minified it? Thanks.

**Comment by eavichay** [comment]
with uglify-js-harmony@2.6.2

thank you for your help, most appreciated.

When can I expect the library to be hosted on cdnjs?

**Comment by PeterDaveHello** [comment]
@eavichay okay so it's uglify-js-harmoy not uglify-js.

**Comment by PeterDaveHello** [comment]
@eavichay already hosted for a while: https://cdnjs.cloudflare.com/ajax/libs/slim.js/2.6.5/Slim.js, just not listed yet because I have question about the minify work :)

**Comment by PeterDaveHello** [comment]
@PeterDaveHello forgot to mention its url: https://cdnjs.com/libraries/slim.js, also online for a while!

**Comment by PeterDaveHello** [comment]
@eavichay please also let me know if you can release the minified file with each tagged version in future, thanks.

**Pull request merged**